File: Assets/Resources/script/GetItem.js
#pragma strict

function OnTriggerEnter (collider : Collider) {
	
	if (collider.tag == "Player") {
		Destroy(this.gameObject);
		return;
	}
	Debug.Log("capsule");
}

function OnCollisionEnter(collision: Collision) {
	Destroy(gameObject);
}

function OnTriggerExit (collider : Collider) {
}

function Start () {

}

function Update () {

}

File: Assets/Resources/script/GetItem.cs
using UnityEngine;
using System.Collections;

public class GetItem : MonoBehaviour {

	// Use this for initialization
	void Start () {
	
	}
	
	// Update is called once per frame
	void Update () {
	
	}

	void OnTriggerEnter(Collider collider){
		if (collider.tag == "Player") {
			if(!unityChanState.IsEquipRight()){
				unityChanState.SetIsEquipRight(true);
			}
			unityChanState.SetWeaponNameRight(this.gameObject.name);
			Destroy(this.gameObject);
			return;
		}

	}

}
